File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1052.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
n = parseInt(lines.shift());

meses = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];

console.log(meses[n-1]);

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1052.cs
using System;

class URI
{
    static void Main() {
        int n = int.Parse(System.Console.ReadLine().Trim());
        string[] meses = {"January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"};
        
        Console.Write(meses[n-1]+"\n");
    }
}